styleguide/_themes/global/scss/forms.scss

```diff
@@ -37,12 +37,12 @@ table {
   .help-block,
   .radio,
   .radio-inline {
-    color: #eb0022;
+    color: $brand-danger;
   }
 
   .form-control {
     background-color: rgba(235, 0, 34, .1);
```

Comment:
While you're at it, can we switch these rgba() values to variable names too, to clean up these sass lint errors? 

(We may need to figure out a naming convention for alpha-based colors as well)